src/detect-engine-analyzer.c

```diff
@@ -861,6 +863,41 @@ static void DumpMatches(RuleAnalyzer *ctx, JsonBuilder *js, const SigMatchData *
                 jb_close(js);
                 break;
             }
+            case DETECT_FLOWBITS: {
+                const DetectFlowbitsData *cd = (const DetectFlowbitsData *)smd->ctx;
+
+                jb_open_object(js, "flowbits");
+                jb_set_uint(js, "variables", cd->or_list_size);
```

Comment:
This will only make sense when this size is not zero (for the ored_flowbits), so maybe we change that name to `ored_flowbits`, move that close to the variables array inside the `else if`, I'd say, so it won't be logged when we only have one flowbit in the rule.